pallets/author-noting/src/test.rs

```diff
@@ -363,3 +363,44 @@ fn test_not_inserting_inherent() {
             assert!(AuthorNoting::latest_author(ParaId::from(1001)).is_none());
         });
 }
+
+#[test]
+fn encode_proof_for_benchmarks() {
```

Comment:
```suggestion
#[test]
#[ignore]
fn encode_proof_for_benchmarks() {
```

We don't need to run this test all the time